PhysicsTools/PatAlgos/plugins/PATIsolatedTrackProducer.cc

```diff
@@ -140,6 +143,10 @@ pat::PATIsolatedTrackProducer::PATIsolatedTrackProducer(const edm::ParameterSet&
                                                       : edm::EDGetTokenT<edm::ValueMap<int>>()),
       usePrecomputedDeDxStrip_(iConfig.getParameter<bool>("usePrecomputedDeDxStrip")),
       usePrecomputedDeDxPixel_(iConfig.getParameter<bool>("usePrecomputedDeDxPixel")),
+      gt2siPixelTrackProbQXY_(consumes<edm::ValueMap<reco::SiPixelTrackProbQXYCollection>>(
```

Comment:
```suggestion
      gt2siPixelTrackProbQXY_(consumes<edm::ValueMap<reco::SiPixelTrackProbQXY>>(
```

Reply:
right, the type is wrong; there are a few examples of using a `ValueMap` in the same file.